src/app.ts

```diff
@@ -44,7 +46,10 @@ export default (
   app.use(addDatasets());
   app.use(jsonld({
     '@language': 'en',
-    ...namespaces,
+    hydra: hydra().value,
+    owl: owl().value,
+    rdf: rdf().value,
+    schema: schema().value,
```

Comment:
@tpluscode Maybe we could add a `base` or `root` function to get the base IRI as a string?

Reply:
I thought about this, and I'm not sure. Do you find yourself often in need of the base URI as string?

It would have to have a unique name, like `schema.__base`. Unless I duplicated the `prefixes` export and generated a default export

```
import prefixes from '@tpluscode/rdf-ns-builders`

app.use(jsonld({
  ...prefixes
}))
```

And neither is great IMO

Reply:
> Do you find yourself often in need of the base URI as string?

Only case I think, as this is outside RDF/JS.

You're right, neither of those is great...

Reply:
Opened https://github.com/zazuko/rdf-vocabularies/issues/55 to try and use a library rather than hard-coding them here.